src/tracks/playabletrack/wavetrack/ui/CutlineHandle.cpp

```diff
@@ -42,36 +42,16 @@ void CutlineHandle::Enter(bool, AudacityProject *)
 #endif
 }
 
-HitTestPreview CutlineHandle::HitPreview(bool cutline, bool unsafe)
+HitTestPreview CutlineHandle::HitPreview(bool unsafe)
 {
    static auto disabledCursor =
       ::MakeCursor(wxCURSOR_NO_ENTRY, DisabledCursorXpm, 16, 16);
    static wxCursor arrowCursor{ wxCURSOR_ARROW };
-   return {
-      (cutline
-       ? XO("Left-Click to expand, Right-Click to remove")
-       : XO("Left-Click to merge clips")),
-      (unsafe
-       ? &*disabledCursor
-       : &arrowCursor)
-   };
+   return { XO("Left-Click to expand, Right-Click to remove"),
+            (unsafe ? &*disabledCursor : &arrowCursor) };
 }
 namespace
 {
-   int FindMergeLine(const WaveTrackLocations &locations,
-      WaveTrack &track, double time)
-   {
-      const double tolerance = 0.5 / track.GetRate();
-      int ii = 0;
-      for (const auto loc: locations) {
-         if (loc.typ == WaveTrackLocation::locationMergePoint &&
-             fabs(time - loc.pos) < tolerance)
-            return ii;
-         ++ii;
-      }
-      return -1;
-   }
```

Comment:
If the issue is ever fully satisfied (and it is not in this PR), then the merging of clips will still be available as a context menu command, and I think you will still need what is deleted here.

Reply:
This is a member function of `CutlineHandle`. The functionality, when presented over the clip menu, will live someplace else. And since it is of the simple sort, I opted for not leaving dead code behind.